R/findpeaks.R

```diff
@@ -170,7 +170,7 @@ findpeaks <- function(data,
   D <- with(expand.grid(A = idx_s, B = t(idx_s)), abs(A - B))
   dim(D) <- c(length(idx_s), length(idx_s))
   diag(D) <- NA                     # eliminate diagonal comparison
-  if (any(D) < MinPeakDistance) {
+  if (isTRUE(any(D < MinPeakDistance))) {
```

Comment:
compare with https://sourceforge.net/p/octave/signal/ci/default/tree/inst/findpeaks.m#l205

Reply:
NB isTRUE handles cases when there are only NA values in D